lib/src/main/java/com/auth0/msg/AbstractMessage.java

```diff
@@ -0,0 +1,180 @@
+package com.auth0.msg;
```

Comment:
I am going to check with Lee what package name we should use.. I don't think we will use com.auth0